tools.js

```diff
@@ -349,13 +358,13 @@ export const registerTools = (
         files: z
           .array(z.string())
           .describe(
-            'Array of absolute file paths to deploy (e.g. ["/home/user/project/src/index.js", "/home/user/project/package.json"])'
+            'Array of absolute file paths to deploy (e.g. "/home/user/project/src/index.js", "/home/user/project/package.json")'
```

Comment:
Why this change? 
Please revert. this is an array, it's relevant to give an array as example